src/cbmc/README.md

```diff
@@ -76,13 +76,15 @@ by other mechanism may categorise their properties differently.
 
 ### Coverage mode
 
-There is one further BMC mode that differs more fundamentally: when `--cover` is
-passed, assertions in the program text are converted into assumptions (these
+There is one further BMC mode that differs more fundamentally: when `--cover`
+is passed, assertions in the program text are converted into assumptions (these
 must hold for control flow to proceed past them, but are not goals for the
-equation solver), while new `assert(false)` statements are added throughout the
-source program representing coverage goals. The equation solving process then
-proceeds the same as in all-properties mode. Coverage solving is implemented by
-`bmc_covert`, but is structurally practically identical to
+equation solver; In cases where this behaviour is undesirable you can pass the
+`--cover-failed-assertions` which makes coverage checking continue even for
```

Comment:
BTW shouldn’t this file be under `doc` rather than `src`? I don’t really understand our documentation structure...

Reply:
The READMEs which are in each module are pulled into here: http://cprover.diffblue.com/folder-walkthrough.html (click on `cbmc`, for example).
We decided to keep them in the source folders so that they are as close as possible to the code.